requirements-test.txt

```diff
@@ -5,7 +5,11 @@
 # Kali Linux, so it makes sense to force the installation of Ansible
 # 2.10 or newer.
 ansible>=2.10,<3
-ansible-lint
+# ansible-lint has a number of breaking changes with the update to v5 as we have
+# documented in https://github.com/cisagov/skeleton-ansible-role/issues/69
+# We can unpin once everything with the v5 update has shaken out and we have a
+# clear path forward.
+ansible-lint<5
 boto3
```

Comment:
Is `boto3` needed?

Reply:
I believe it's for https://github.com/cisagov/ansible-role-cyhy-commander/blob/2a59c7a94bec0443615ee65c05474df1440f8066/defaults/main.yml#L3

Reply:
Aha!  Thank you, I am blind.